src/components/Todos/Todos.tsx

```diff
@@ -0,0 +1,127 @@
+import React, { useState } from 'react';
+import classNames from 'classnames';
+import { CSSTransition, TransitionGroup } from 'react-transition-group';
+import { Todo } from '../../types/Todo';
+import { TempTodo } from '../TempTodo';
+
+type Props = {
+  todos: Todo[] | null;
+  newTodoTitle: string;
+  isTodoAdded: boolean;
+  removeTodo: (targetId: number) => void;
+  targetTodosIds: number[];
+  changeTodoStatus: (todoId: number, status: boolean) => void;
+  callEditeMode: (title: string) => void;
+  updatingTitle: string;
+  setUdatingTitle: (value: string) => void;
+  submitTitleUpdating: (id: number, oldTitle: string) => void;
+};
+
+export const Todos: React.FC<Props> = ({
+  todos,
+  newTodoTitle,
+  isTodoAdded,
+  removeTodo,
+  targetTodosIds,
+  changeTodoStatus,
+  callEditeMode,
+  updatingTitle,
+  setUdatingTitle,
+  submitTitleUpdating,
+}) => {
+  const [editedTodoId, setEditedTodoId] = useState(0);
```

Comment:
Correctly to set null, if you have a todo with id 0, will the editing input be immediately visible?